src/plot.cpp

```diff
@@ -405,28 +454,21 @@ void Plot::set_default_colors(pugi::xml_node plot_node)
   }
 }
 
-void Plot::set_user_colors(pugi::xml_node plot_node)
+void PlottableInterface::set_user_colors(pugi::xml_node plot_node)
 {
-  if (!plot_node.select_nodes("color").empty() && PlotType::voxel == type_) {
-    if (mpi::master) {
-      warning(
-        fmt::format("Color specifications ignored in voxel plot {}", id_));
-    }
-  }
-
```

Comment:
I deleted these because it's hard to check this in the base class. I'd rather just see the colors not get used. Also, the consequence of not getting warned about this is minimal--was there ever any expectation that colors would carry through to a voxel plot?

Reply:
No it wasn't an expectation, but it was a nice clarification for users so they don't expect their color specs to show up in paraview/visit or something like that.

Reply:
I think we could wrap the `set_user_color` call in the `SlicePlotBase` class to keep this check though, no?